src/pages/subjects/components/show/SubjectTable.tsx

```diff
@@ -150,6 +238,33 @@ const SubjectTable = () => {
                         ) : (
                             <MenuItem value={undefined}>No Data</MenuItem>
                         )}
+                        {
+                            <Tooltip title="Add Semester">
+                                <MenuItem
+                                    onClick={() => {
+                                        setAddSemester({
+                                            ...addSemester,
+                                            open: true,
+                                            semEnable: true,
+                                            record: semesterData,
+                                        });
+                                    }}
+                                >
+                                    <AddIcon
+                                        style={{
```

Comment:
is there no way to avoid giving this custom styling.
Also please use mui theme variables. dont hardcode the colors. use primary secondary